client/lxd_instances.go

```diff
@@ -2403,7 +2410,13 @@ func (r *ProtocolLXD) ConsoleInstance(instanceName string, console api.InstanceC
 	}
 
 	// Send the request
-	op, _, err := r.queryOperation("POST", fmt.Sprintf("%s/%s/console", path, url.PathEscape(instanceName)), console, "", false)
+	var op Operation
+	if r.CheckExtension("client_operation_wait") == nil {
```

Comment:
Just a small nit, but since the error is ignored here, I think `r.HasExtension("client_operation_wait")` would be cleaner.

Maybe even something like this?
```go
useEventListener := r.HasExtension("client_operation_wait")
op, _, err = r.queryOperation("POST", fmt.Sprintf("%s/%s/console", path, url.PathEscape(instanceName)), console, "", useEventListener)
```

Reply:
I didn't want to use `HasExtension` since it has been deprecated but I am happy either way really.

Reply:
Oh, haven't realized that before. Then LGTM 👍